src/models/objectStore.js

```diff
@@ -0,0 +1,86 @@
+import { listObjectStores, getObjectStore, createObjectStore, deleteObjectStore } from '../services/objectStore'
+import { wsChanges, updateState } from '../utils/websocket'
+import queryString from 'query-string'
+import { enableQueryData } from '../utils/dataDependency'
+
+export default {
+  namespace: 'objectstorage',
+  state: {
+    ws: null,
+    data: [],
+    selected: {},
+    resourceType: 'objectstore',
+    socketStatus: 'closed',
+  },
+  subscriptions: {
+    setup({ dispatch, history }) {
+      history.listen(location => {
+        if (enableQueryData(location.pathname, 'objectstore')) {
+          dispatch({
+            type: 'query',
+            payload: location.pathname.startsWith('/objectstore') ? queryString.parse(location.search) : {},
+          })
+        }
+      })
+    },
+  },
+  effects: {
+    *query({ payload }, { call, put }) {
+      const data = yield call(listObjectStores, payload)
+      yield put({ type: 'listObjectStores', payload: { ...data } })
```

Comment:
Is a `put` used by reason here? I assume it should be `get` instead.

Reply:
It should be a `put`. `get` would cause errors